mantidproject/mantid
# Observe peak parameters in GetDetectorOffset

**Description of work.**

Pulls functions related to estimating peak parameters out of `FitPeaks` so that they can be reused in the `GetDetectorOffset` algorithm. Refer to [PD262](https://code.ornl.gov/sns-hfir-scse/diffraction/powder/powder-diffraction/-/issues/262).




**To test:**

Run the unit tests for `FitPeaks` and `GetDetectorOffsets` to make sure refactoring did not change tests.








---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
